cg/test-key.json

```diff
@@ -0,0 +1 @@
+{"address":"54dac00c8d5dae1e6ab7bb1628ec142a87a90615","crypto":{"cipher":"aes-128-ctr","ciphertext":"05a067984c13972c4cccc2b0ba531d4c43082764ef72db2b7c74374db6378de4","cipherparams":{"iv":"1d5f61c82dd1a0e45d4b8266ef3d9c9f"},"kdf":"scrypt","kdfparams":{"dklen":32,"n":262144,"p":1,"r":8,"salt":"7aef50f3b24228e2ac02e6d70ec64ccaeb9b83a2a7b901dcddb762e6529c2f68"},"mac":"a75ece204d7b9237920160f6ce9334381ba1a597f190194a3055235cf6a26183"},"id":"22e3fd83-b104-4d8c-836c-4baf749724c9","version":3}
```

Comment:
This file should not be commited to the repo. Don't know how it got there – should be deleted with the scripts.